source/code/plugin/out_mdm.rb

```diff
@@ -167,10 +168,12 @@ def send_to_mdm(post_body)
           # Not raising exception, as that will cause retries to happen
         elsif !response.code.empty? && response.code.start_with?("4")
           # Log 400 errors and continue
+          ApplicationInsightsUtility.sendCustomEvent("AKSCustomMetricsMDMPostError-#{response.code} ", {})
```

Comment:
Can you remove this ? We don't want to log 403's as they will be too many (even one from every node every 60 mins)

Reply:
we specifically look for 403 in the block above this. So these wont log 403's

Reply:
what is the exception you log in line #167 ? It is inside the 403 check block.

Reply:
LIne 167 is an exception, which was there before. I removed it now. And Added the response code and error as dimensions with a single fixed custom Event Name "AKSCustomMetricsMDMPostError"